server/middleware/Validator.js

```diff
@@ -0,0 +1,17 @@
+class Validator{
+    static validateString(string){
+        if(!string || string === undefined || string.toString().trim() === '' || typeof string !== 'string') return false;
+        if(string.lenght < 3 || string.lenght > 500) return false;
+        const stringAllowed = /^[a-zA-Z-'\s\d]+$/;
+        if(!string.match(stringAllowed)) return false;
+        return true;
+    }
+
+    static validateAddress(string){
+        if(!string || string === undefined || string.toString().trim() === '' || typeof string !== 'string') return false;
```

Comment:
Confirm if you can add this if statement in one function, then call it when you want to use it.

Reply:
okay, i will do that